src/styles/_catalog.scss

```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
+    transition: transform 300ms;
```

Comment:
```suggestion
    transition: transform 300ms, color 300ms;
```

Reply:
you change 2 properties, but add transition for only one of them

Reply:
Thank you, fixed